src/rdkafka_request.c

```diff
@@ -1240,17 +1296,26 @@ void rd_kafka_JoinGroupRequest (rd_kafka_broker_t *rkb,
 void rd_kafka_LeaveGroupRequest (rd_kafka_broker_t *rkb,
                                  const rd_kafkap_str_t *group_id,
                                  const rd_kafkap_str_t *member_id,
+                                 const rd_kafkap_str_t *group_instance_id,
                                  rd_kafka_replyq_t replyq,
                                  rd_kafka_resp_cb_t *resp_cb,
                                  void *opaque) {
         rd_kafka_buf_t *rkbuf;
+        int16_t ApiVersion = 0;
+        int features;
+
+        ApiVersion = rd_kafka_broker_ApiVersion_supported(rkb,
+                                                          RD_KAFKAP_LeaveGroup,
+                                                          0, 1,
+                                                          &features);
 
         rkbuf = rd_kafka_buf_new_request(rkb, RD_KAFKAP_LeaveGroup,
                                          1,
                                          RD_KAFKAP_STR_SIZE(group_id) +
                                          RD_KAFKAP_STR_SIZE(member_id));
         rd_kafka_buf_write_kstr(rkbuf, group_id);
         rd_kafka_buf_write_kstr(rkbuf, member_id);
+        rd_kafka_buf_ApiVersion_set(rkbuf, ApiVersion, 0);
```

Comment:
Add an empty line above this call to separate the buf writers from the apiversion, for consistency.